apps/meteor/client/views/root/AppLayout.tsx

```diff
@@ -52,8 +54,9 @@ const AppLayout = () => {
 	useTokenPassAuth();
 	useCustomOAuth();
 	useCorsSSLConfig();
-
 	useOTRMessaging();
+	useFetchCannedResponses();
+	useCannedResponsesStream();
```

Comment:
please merge both hooks I dont think we gonna use them separated

Reply:
We may not use them separately, but having those two things inside the same useEffect is a separation of concerns issue. Also I can see a future in which we may use `useFetchCannedResponses` for other parts of the system.

Reply:
ok, so please create an `useCannedResponse` that uses `useFetchCannedResponse` gets the data result and adds to the Collection, meanwhile starts a stream, but the useStream alone is not useful.

I never said to put both them inside the same **useEffect**, but inside the useCannedResponsesStream something like`useCannedResponse`

notice that both hooks cause a mutation on the same collection, so both share one same concern: fulfill the CannedResponse Collection.

if somewhere people wants to use the `useFetchCannedResponses` to handle the data from the server they should not mutate the Cached collection again